modules/lvgl/Kconfig.memory

```diff
@@ -52,6 +52,22 @@ config LV_Z_MEMORY_POOL_CUSTOM_SECTION
 	  memory pool to a custom location, such as tightly coupled or
 	  external memory.
 
+config LV_Z_MEMORY_POOL_CUSTOM_GENERIC_SECTION
+	bool "Link memory pool to custom generic section"
+	depends on LV_Z_MEM_POOL_SYS_HEAP
+	help
+	  Place LVGL memory pool in custom generic section automatically
+	  created by Zephyr when compatible zephyr,memory-region is used.
+	  LV_Z_MEMORY_POOL_CUSTOM_SECTION must be disabled in order to use
+	  generic section.
+
+config LV_Z_MEMORY_POOL_CUSTOM_GENERIC_SECTION_NAME
```

Comment:
Dont know about the naming of the symbol, "custom generic" seems to contradict itsself a bit imo. Maybe something involving the memory region concept?